pandas/core/indexing.py

```diff
@@ -1920,11 +1920,35 @@ def _convert_key(self, key, is_setter=False):
 
 
 class _iAtIndexer(_ScalarAccessIndexer):
-    """Fast integer location scalar accessor.
+    """
+    Selects a single value for a row/column pair by integer position.
 
-    Similarly to ``iloc``, ``iat`` provides **integer** based lookups.
-    You can also set using these indexers.
+    Useful if performance is a major concern and you only need to
+    get or set a value at a particular row/column.
 
+    See Also
+    --------
+    at : Selects a single value for a row/column label pair
+    loc : Selects a group of rows and columns by label(s)
+    iloc : Selects group of rows and columns by integer position(s)
+
+    Returns
+    -------
+        None
+
+    Examples
+    --------
+    >>> df = pd.DataFrame([[0,2,3], [0,4,1], [10,20,30]])
```

Comment:
pep8: spaces after comma.

Reply:
Also, can you specify string column names? eg `columns=['A', 'B', 'C']` 
to make it clear that it is position based (with the integer indexes it is dubious)